### PR TITLE
Bl orient 809

### PR DESCRIPTION
**Pull request by DariaElagina**

### CODE REVIEW

File: LondonBritishLibrary/orient/BLorient809.xml [hunk 1]
```diff
@@ -104,13 +81,467 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                                the contect is partially damaged through water stains.</condition>
+                              </supportDesc>
+                          </objectDesc>
+                          <additions>
+                            <list>
+                              <item xml:id="a2">
+                                <locus target="#4v"></locus>
+                                <desc type="GuestText">
+                                  An extract from Mār Barsā</desc>
+                            <q xml:lang="gez">ይቤ፡ መር፡ በርሳ፡ እምጥንተ፡ በአቶሙ፡ ለእስራኤል፡ በ፸ ነፍስ፡ እስከ፡
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient809.xml:167]: ማር or መር?

File: LondonBritishLibrary/orient/BLorient809.xml [hunk 2]
```diff
@@ -104,13 +81,467 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                                  of Japhet, Shem and Ham.
+                                  <q xml:lang="gez">ውሉደ፡ ያፌት፡ ፲ወ፭ ነገድ፡ ወመክፈልቶሙ፡ ኮነ፡ ምድረ፡ ፀጋም፡ ወመንፈቀ፡
+                                  ምዕራብ፡ ወውሉደ፡ ሴምኒ፡ ፳ወ፭፡ ነገድ፡ ወመክፈልቶሙ፡ ኮነ፡ ምድረ፡ ጽባሕ፡ ወመንፈቀ፡ አዜብ፡ ወደቂቀ፡
+                                ካምኒ፡ ፴ወወ፪ ነገድ፡ ወመክፈልቶሙ፡ ኮነ፡ ምድረ፡ አዜብ፡ ወመንፈቀ፡ ምዕራብ፤ ወኮኑ፡ ኵሉ፡ ውሉደ፡ ኖኅ፡
+                              ፸ወ፪፡ ወእንበይነ፡ ዝንቱ፡ ተከፍሉ፡ ኀበ፡ ፸ወ፪፡ ልሳናት፡</q>
+                                </desc>
+                              </item>
+                              <item xml:id="a4">
+                                <locus target="#4v"></locus>
+                                <desc type="GuestText">Short glossary.</desc>
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient809.xml:182]: Maybe you could add, in Amharic and Geez?

File: LondonBritishLibrary/orient/BLorient809.xml [hunk 3]
```diff
@@ -104,13 +81,467 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                      </physDesc>
+                    </msPart>
+                    <msPart xml:id="p3">
+                      <msIdentifier></msIdentifier>
+                      <msContents>
+                        <summary></summary>
+                        <msItem xml:id="p3_i1">
+                          <locus from="7ra" to="123v"></locus>
+                          <title type="incomplete" ref="LIT1011Abusak"></title>
+                          <textLang mainLang="gez"></textLang>
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient809.xml:215]: maybe add otherLangs Amharic, if there are Amharic parts throughout? Or do you think they constitute a later addition?

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you!

File: LondonBritishLibrary/orient/BLorient809.xml (hunk 3 above)
Comment (DariaElagina) [reply LondonBritishLibrary/orient/BLorient809.xml:215]: I think those Amharic commentaries are of a later origin, not attested in other mss as well

**Review by thea-m — APPROVED** [review]

**Pull request merged**